config/config.go

```diff
@@ -151,9 +151,5 @@ func (cfg *CLIConfig) VerifyAuthentication() error {
 		return errors.New("You are not logged into DeepSource. Run \"deepsource auth login\" to authenticate.")
 	}
 
-	// Check if the token has already expired
```

Comment:
Lets comment it for now and not delete this snippet since it may be required once we store the `TokenExpiresIn` field in the config as well.

Reply:
I had the same thought as well, but I was hesitant to comment it out. Will do the changes! 😄